osidb/api_views.py

```diff
@@ -1046,3 +1048,16 @@ def options(self, request, *args, **kwargs):
         }
         # Return a response with the allowed methods and headers
         return Response(status=200, headers=headers)
+
+
+class AuditView(ModelViewSet):
+    """basic view of audit history events"""
+
+    queryset = pghistory.models.Events.objects.all().order_by("-pgh_created_at")
+    serializer_class = AuditSerializer
+    filter_backends = (DjangoFilterBackend,)
+    filterset_fields = ["pgh_slug", "pgh_label", "pgh_obj_model", "pgh_created_at"]
+    http_method_names = get_valid_http_methods(
+        ModelViewSet, excluded=["delete", "post", "update"]
+    )
+    permission_classes = [IsAuthenticatedOrReadOnly]
```

Comment:
I have a question (maybe I already asked in the past). Is the audit/history safeguarded by the same ACLs? I do not understand this part well but it either should be or the audit would need to be only for the privileged users not to expose anything above the ACLs.

Reply:
I see below in the migration that the audit table has `acl_read`/`acl_write` so I think my question is probably unnecessary and it seem everything is as it should be.